src/Bookings.js

```diff
@@ -1,18 +1,77 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [customerId, setCustomerId] = useState(null);
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [errorOccurred, setErrorOccurred] = useState({
+    ok: true,
+    status: "",
+    statusText: ""
+  });
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`)
+      .then(res => {
+        if (!res.ok) {
+          setErrorOccurred({
+            ok: res.ok,
+            status: res.status,
+            statusText: res.statusText
+          });
+          throw new Error(res);
+        } else {
+          return res.json();
+        }
+      })
+      .then(data => {
+        setBookings(data);
+        setData(data);
+        setIsLoading(false);
+      })
+      .catch(error => {
+        console.log(error);
```

Comment:
Anything else that we could do in our catch function, would the user be told if the request failed like this?